### PR TITLE
Manpages

### PR DESCRIPTION
**Pull request by rillian**
Quick update to fix two manpage issues I noticed in #25689.

### CODE REVIEW

File: man/rustc.1 [hunk 1]
```diff
@@ -1,4 +1,4 @@
-.TH RUSTC "1" "March 2014" "rustc 0.13.0" "User Commands"
+.TH RUSTC "1" "May 2015" "rustc 1.2.0" "User Commands"
```
Comment (steveklabnik) [inline man/rustc.1]: 1.2 will be released in july

File: man/rustdoc.1 [hunk 2]
```diff
@@ -1,4 +1,4 @@
-.TH RUSTDOC "1" "March 2014" "rustdoc 0.13.0" "User Commands"
+.TH RUSTDOC "1" "May 2015" "rustdoc 1.2.0" "User Commands"
```
Comment (steveklabnik) [inline man/rustdoc.1]: same

**Comment by rillian** [comment]
ITYM August r? @steveklabnik

**Comment by alexcrichton** [comment]
@bors: r+ bb61b0b8bb7c078bfa035cd5c392f029802b49ec rollup

Thanks!

**Comment by tshepang** [comment]
I always thought the date on manpages was "last edited", not "expected release date" or "release date of version this applies to". I think "last edited" is also better because some stuff may change between now and 1.2 release.

BTW, do we need to mention the versions? We are going to have to remember updating them every release, even though manpage content has not changed.

**Comment by rillian** [comment]
> I always thought the date on manpages was "last edited", not "expected release date". I think "last edited" is also better because some stuff may change between now and 1.2 release.

It is. The "last edited" and "release date" should also correspond, so I think the question is which of those you think is more likely...

> BTW, do we need to mention the versions? We are going to have to remember updating them every release, even though manpage content has not changed.

Not strictly. Many manpages have only a modification date. However, it's useful to know what release the page refers to, so unless we can't maintain them at all, I think the version number should stay.

The real fix to to automate the version and date bump.

**Comment by tshepang** [comment]
@rillian I don't see a reason why "last edited" and "release date" should correspond. It's misleading. It would be accurate to state the actual edit date, and just leave the version at **1.2.0-dev**, and only update that version just before release (or during Beta) to **1.2**, after verifying that the manpage is still accurate. So, the version should only be updated if someone verifies its accuracy... it should not be automated.

**Pull request merged**